commands/hubdb/delete.ts

```diff
@@ -1,37 +1,41 @@
-// @ts-nocheck
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const { logError } = require('../../lib/errorHandlers/index');
-const { deleteTable } = require('@hubspot/local-dev-lib/api/hubdb');
-const { trackCommandUsage } = require('../../lib/usageTracking');
-const {
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { logError } from '../../lib/errorHandlers/index';
+import { deleteTable } from '@hubspot/local-dev-lib/api/hubdb';
+import { trackCommandUsage } from '../../lib/usageTracking';
+import {
   addConfigOptions,
   addAccountOptions,
   addUseEnvironmentOptions,
-} = require('../../lib/commonOpts');
-const {
-  selectHubDBTablePrompt,
-} = require('../../lib/prompts/selectHubDBTablePrompt');
-const { promptUser } = require('../../lib/prompts/promptUtils');
-const { EXIT_CODES } = require('../../lib/enums/exitCodes');
-const { i18n } = require('../../lib/lang');
+} from '../../lib/commonOpts';
+import { selectHubDBTablePrompt } from '../../lib/prompts/selectHubDBTablePrompt';
+import { promptUser } from '../../lib/prompts/promptUtils';
+import { EXIT_CODES } from '../../lib/enums/exitCodes';
+import { i18n } from '../../lib/lang';
+import { CommonArgs, ConfigArgs } from '../../types/Yargs';
 
 const i18nKey = 'commands.hubdb.subcommands.delete';
 
-exports.command = 'delete [table-id]';
-exports.describe = i18n(`${i18nKey}.describe`);
+export const command = 'delete [table-id]';
+export const describe = i18n(`${i18nKey}.describe`);
 
-exports.handler = async options => {
-  const { force, derivedAccountId } = options;
+type HubdbDeleteArgs = CommonArgs &
+  ConfigArgs & { tableId?: number; dest?: string };
```

Comment:
I don't think dest is used anywhere in this command, maybe just snuck its way in from copy/paste?